client/extensions/woocommerce/app/order/order-details/row-refund.js

```diff
@@ -30,10 +31,10 @@ class OrderRefundRow extends Component {
 		}
 
 		return (
-			<div className="order__details-total-refund">
-				<div className="order__details-totals-label">{ translate( 'Refunded' ) }</div>
-				{ showTax && <div className="order__details-totals-tax"></div> }
-				<div className="order__details-totals-value">
+			<div className="order-details__total-refund">
+				<div className="order-details__totals-label">{ translate( 'Refunded' ) }</div>
```

Comment:
Hi! I've found a possible matching string that has already been translated **37** times:
`translate( 'Refunded To' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/store--order-file-cleanup)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).